### PR TITLE
Modifications on Transfer Learning Example

### PR DESCRIPTION
**Pull request by charlieJ107**
Modified Tensorflow Transfer Leaning example from stock tensorflow to bigdl nano.

### CODE REVIEW

**Comment by charlieJ107** [comment]
Jekins test passed at 
http://10.112.231.51:18888/job/ZOO-PR-NanoTests/10/

**Comment by yangw1234** [comment]
need to add a readme

File: python/nano/example/tensorflow/transfer_learning/Readme.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# BigDL-Nano Transfer Learning Example with Tensorflow
+
+This example describes how BigDL Nano optimizes transfer learning.
```
Comment (yangw1234) [inline python/nano/example/tensorflow/transfer_learning/Readme.md:3]: how about `how to use BigDL Nano to optimize a tensorflow transfer learning workload`?

File: python/nano/example/tensorflow/transfer_learning/Readme.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# BigDL-Nano Transfer Learning Example with Tensorflow
+
+This example describes how BigDL Nano optimizes transfer learning. 
+We use cat-vs-dog data sets to train a partial freezen neural network based on Mobile Net V2, and train unfreezen trainable variables through transfer learning.
```
Comment (yangw1234) [inline python/nano/example/tensorflow/transfer_learning/Readme.md:4]: `the cat-vs-dog dataset` not `cat-vs-dog data sets`?
`partially  frozen` not `partial freeze`
`MobileNetV2` not `Mobile Net V2`
`unfrozen` not `unfreezen`

Comment (yangw1234) [reply python/nano/example/tensorflow/transfer_learning/Readme.md:4]: I think we can simply remove this line.

File: python/nano/example/tensorflow/transfer_learning/Readme.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+This example is migrated from the tensorflow tutorial notebook at 
+https://github.com/tensorflow/docs/blob/r2.4/site/en/tutorials/images/transfer_learning.ipynb
+
+## Quick Start
+1. Prepare Envrionment
```
Comment (yangw1234) [inline python/nano/example/tensorflow/transfer_learning/Readme.md:10]: `Environment` not `Envrionment`

File: python/nano/example/tensorflow/transfer_learning/Readme.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+2. Run the Example
+
+    You can run this example in your conda environment with the following command:
+    ```
+    python  transfer_learning.py
+    ```
+
+
+## Workflow and Results
+We use the Mobile Net V2 as our base model, build our model combined with Preprocessing, Prediction and other Layers. The initial loss and accuary will be printed before training steps like this:
```
Comment (yangw1234) [inline python/nano/example/tensorflow/transfer_learning/Readme.md:33]: `We use the MobileNetV2 network as our base model combined with "Preprocessing" and "Prediction" Layers`

File: python/nano/example/tensorflow/transfer_learning/Readme.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+```
+Number of trainable variables: 2
+26/26 [==============================] - 5s 87ms/step - loss: 0.9020 - accuracy: 0.4468
+initial loss: 0.90
+initial accuracy: 0.45
+```
+
+Then we freeze the base model and train other layers in the model. After 10 epoches of training, We unfreeze the last 100 layers of base model and continued to train on the basis of previous training. We will get the final accuracy after the evaluation of the model at the end of workflow.
```
Comment (yangw1234) [inline python/nano/example/tensorflow/transfer_learning/Readme.md:42]: I think the second sentence should be `After 10 epoches of training, we unfreeze the model except the first 100 layers of the base model and continue to train.`

File: python/nano/example/tensorflow/transfer_learning/Readme.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+```
+Number of trainable variables now: 56
+Epoch 10/20
+63/63 [==============================] - 14s 172ms/step - loss: 0.1472 - accuracy: 0.9390 - val_loss: 0.0623 - val_accuracy: 0.9790
+Epoch 11/20
+63/63 [==============================] - 10s 149ms/step - loss: 0.1188 - accuracy: 0.9480 - val_loss: 0.0571 - val_accuracy: 0.9802
+...
+```
+
+After the training, we will evaluate the model. You can check the accuracy at the end of workflow.
```
Comment (yangw1234) [inline python/nano/example/tensorflow/transfer_learning/Readme.md:53]: How about `After the training, we evaluate the model.` (You are using the simple present tense to decribe steps the whole time right? Why switch to the future tense.)

**Comment by yangw1234** [comment]
pr validation should be run again if you made some changes to the code

**Comment by yangw1234** [comment]
LGTM

**Comment by charlieJ107** [comment]
Jekins Passed in http://10.112.231.51:18888/job/ZOO-PR-NanoTests/13/
